README.md

```diff
@@ -1,96 +1,118 @@
+If you're new here, here are some things you should know:
+ - We actively curate issues and try to make them as self-contained as possible for people new to the application; those ones are tagged "Help Wanted"
+ - We're actively watching for Pull Requests and you shouldn't have to wait very long for a review. Try to make sure your build passes (`rubocop -a` is a frequent need) and that you've addressed the requirements in the issue
+ - There is a wiki article called [Application Overview](https://github.com/rubyforgood/human-essentials/wiki/Application-Overview). It needs a tiny bit of updating, but is mostly still accurate. It will introduce you to some vocabulary and general concepts, if you find something confusing and want to find the answer on your own.
+ - Check the [`CONTRIBUTING.md`](CONTRIBUTING.md) file for a guide on how to get started
+ - This is a 100% volunteer-supported project, please be patient with your correspondence. We do handle issues and PRs with more fervor during Hacktoberfest & Conferences, but most (all?) of us have day jobs and so responses to questions / pending PRs may not be immediate. Please be patient, we'll get to you! :)
 
-#### Origins
+#### 💬 Join us on slack
+Please feel free to join us on Slack!
+You can sign up at https://rubyforgood.herokuapp.com and find us in #human-essentials.
 
-This project took what we built for the [Portland Diaper Bank in 2016](https://github.com/rubyforgood/pdx_diaper) and turned it into a multitenant application, something that all essentials banks can use. We re-used models, code and other documentation where applicable as well as implemented new features and functionality requested by the prime stakeholder (PDXDB). We're super excited to have had Rachel Alston, the director of the Portland Diaper Bank, attending our event in 2017, providing guidance and giving us the best chance of success!
+There are numerous other folks that can chime in and answer questions -- please ask and someone will probably be there to help!
 
-### Development
+## 🛠️ Getting Started
 
-#### Installation Instructions
+### Install Ruby
+You will need to first install the required ruby version specified in the [`.ruby-version`](.ruby-version) file. GoRails has a very detail oriented guide for installing ruby on Ubuntu, Windows, and macOSX. You can check out that guide [here](https://gorails.com/setup/osx/12-monterey). **Follow only the Installing Ruby step, as our project setup differs**
 
-The `installation.md` file ([https://github.com/rubyforgood/human-essentials/blob/main/installation.md](https://github.com/rubyforgood/human-essentials/blob/main/installation.md)) has detailed instructions for installation and configuration of an Ubuntu host to run this software. Although there is not a document for Mac OS, it may be helpful for that as well.
 
-#### Ruby Version
+It is highly recommended you use a ruby version manager such as:
```

Comment:
I wonder if this section isn't a bit confusing to newer folks. The gorails guide above has you install rbenv above. So if you're new and following the guide above, you're fine and this irrelevant information. If you're experienced you understand the importance of a version manager and already have one that you like and so pointing out options probably isn't necessary. 

Additionally, if I'm new you've given me three options, but not told me how to make a choice. At a minimum, it would be good to say that rbenv is the preferred option or the one that should be picked unless you have a reason to pick the other ones. 

Removing this section completely makes the guide slightly shorter, and I don't think it makes it any less informative.